apps/web/package.json

```diff
@@ -73,6 +73,7 @@
     "handlebars": "^4.7.7",
     "ical.js": "^1.4.0",
     "ics": "^2.37.0",
+    "jose": "^4.11.2",
```

Comment:
Used to encrypt and decrypt the JWT the way `next-auth` does it.

Reply:
![Ill Allow It Spanish GIF](https://media3.giphy.com/media/NCjISbEPFxm48/200.gif)